Analysis/bin/CreateSpectralHists.cxx

```diff
@@ -24,6 +25,8 @@ struct Arguments {
     run::Argument<std::string> exclude_list{"exclude-list", "comma separated list of files to exclude", ""};
     run::Argument<std::string> exclude_dir_list{"exclude-dir-list",
                                                "comma separated list of directories to exclude", ""};
+    run::Argument<std::string> mode{"mode",
```

Comment:
can we run for all modes in one go?

Reply:
No, at the moment I did not add this functionality, for instance DisplacedTau definition does not valid for any analysis but would just require more time to create spectrum hists and also initializing of some extra variables. I propose every analysis carefully checks its own definition.